codingStandards/CodingStandard-CSharp.md

```diff
@@ -0,0 +1,367 @@
+# C# Coding Standard
+
+**Important:** Use [CSharpGuidelines](https://github.com/dennisdoomen/CSharpGuidelines) for any topics not covered in this document.
```

Comment:
Any reason to choose this one over the one from Microsoft? Is this one different from the Microsoft one?